metricbeat/docs/index.asciidoc

```diff
@@ -1,6 +1,7 @@
 = Metricbeat Reference
 
 :libbeat-dir: ../../libbeat
+:libbeat-xpack-dir: ../../x-pack/libbeat
```

Comment:
Did this build OK for you when you built the docs locally? I had to use `../../../x-pack/libbeat`.

Reply:
Yes you are right, this fixed it. Thanks!